packages/theme/example/index.html

```diff
@@ -1054,6 +1060,7 @@ <h1 id="forms">Forms</h1>
 
                 <div class="form-group">
                   <select class="form-control" id="selectDefault">
+                    <option disabled default selected>Select an option</option>
```

Comment:
Dunno `default` attr, what's its purpose?

Reply:
To identify the placeholder to remove from the dropdown list when selecting a value, also we can't use disabled attribute as we may have real disabled values in the dropdown

Reply:
removed the default attribute